cmd/acr/purge_test.go

```diff
@@ -1021,19 +1037,20 @@ var (
 		ImageName:      &testRepo,
 		TagsAttributes: nil,
 	}
-	tagName                = "latest"
-	digest                 = "sha:abc"
-	multiArchDigest        = "sha:356"
-	deleteEnabled          = true
-	deleteDisabled         = false
-	writeEnabled           = true
-	writeDisabled          = false
-	lastUpdateTime         = time.Now().Add(-15 * time.Minute).UTC().Format(time.RFC3339Nano) //Creation time -15minutes from current time
-	lastUpdateTime1DayAgo  = time.Now().Add(-24 * time.Hour).UTC().Format(time.RFC3339Nano)   //Creation time -1d from current time
-	lastUpdateTime2DaysAgo = time.Now().Add(-48 * time.Hour).UTC().Format(time.RFC3339Nano)   //Creation time -2d from current time
-	lastUpdateTime3DaysAgo = time.Now().Add(-72 * time.Hour).UTC().Format(time.RFC3339Nano)   //Creation time -3d from current time
-	invalidLastUpdateTime  = "date"
-	OneTagResult           = &acr.RepositoryTagsType{
+	tagName                    = "latest"
+	digest                     = "sha:abc"
```

Comment:
Nit. Probably best to generate a full digest for the sake of consistency.

Reply:
done.

Reply:
I am getting `Potential hardcoded credentials (gosec)` after I generated random sha256 value for digests.

Reply:
Can we add a suppression comment? (#nosec) https://github.com/securego/gosec#annotating-code